src/Card/CardSelect.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Card from 'react-bootstrap/Card';
+import Form from '../Form';
+
+const CardSelect = ({
+  className,
+  cardsData,
+  onChange,
+  selectedCardName,
+}) => {
+  const [currentRecord, setCurrentRecord] = useState(null);
+  useEffect(() => {
+    setCurrentRecord(selectedCardName);
+  }, []);
+
+  return (
+    <>
+      <Form.Group>
+        <Form.RadioSet
+          name="radio-select-card"
+          onChange={(e) => {
+            setCurrentRecord(e.target.value);
+            onChange(e.target.value);
+          }}
+          defaultValue={selectedCardName}
+          isInline
+          className="row"
+        >
+          {cardsData?.map(data => (
+            <div
+              key={data.name}
+              className={className}
+              role="group"
+              aria-label={data.title}
+            >
+              <Card className={currentRecord === data.name ? 'border border-dark' : null}>
+                <Card.Img variant="top" src={data.img} />
+                <Card.Body>
+                  <Card.Title as="h5" className="card-title mb-1">
+                    <label htmlFor={data.name}>{data.title}</label>
+                    <div className="float-right"><Form.Radio id={data.name} value={data.name} /></div>
+                  </Card.Title>
+                  <Card.Text>
+                    <>{data.text}</><br />
```

Comment:
Another idea. We can pass an array of dicts with className and text. With this card-text elements can be added dynamically and as many as required and with classes that can be easily styled.
```
<Card.Text>
textElements?.map(ele => (
<span class={ele.key}>{ele.text}</span>
))
</Card.Text>
```

Reply:
Done